sdks/java/javadoc/pom.xml

```diff
@@ -84,16 +89,6 @@
 
     <dependency>
       <groupId>org.apache.beam</groupId>
-      <artifactId>beam-sdks-java-io-google-cloud-platform</artifactId>
```

Comment:
This and beam-sdks-java-core were repeated (they appear twice). Look at the final file, both are still there.